libs/transloco/src/lib/scope-resolver.ts

```diff
@@ -1,10 +1,10 @@
-import { TranslocoScope, ProviderScope, MaybeArray } from './types';
+import { TranslocoScope, ProviderScope } from './types';
 import { TranslocoService } from './transloco.service';
 import { isScopeObject, toCamelCase } from './helpers';
 
 type ScopeResolverParams = {
   inline: string | undefined;
-  provider: MaybeArray<TranslocoScope>;
+  provider: TranslocoScope;
```

Comment:
I believe this was done unintentionally, scope resolver is not able to process array of scopes. There's no compilation error only because of `isScopeObject` type guard, that is also not supposed to handle array of scopes